data-otservbr-global/npc/kroox.lua

```diff
@@ -66,8 +66,8 @@ local function creatureSayCallback(npc, creature, type, message)
 			player:setStorageValue(Storage.SamsOldBackpack, 2)
 			player:setStorageValue(Storage.SamsOldBackpackDoor, 1)
 		end
-	elseif MsgContains(message, "measurements") then
-		if player:getStorageValue(Storage.Postman.Mission07) >= 1 and player:getStorageValue(Storage.Postman.MeasurementsKroox) ~= 1 then
+	elseif MsgContains(message, "lokurs measurements") then
```

Comment:
contains `measurements` it's correctly. if you want to check the real word, use `if message == "lokurs measurements"`